### DISCUSSION TITLE
'batch_uuid' Column not found

### DISCUSSION THREAD

**Discussion by Soltee** [spatie/laravel-activitylog#1026] [Q&A]
/********** ERROR*****************/
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'batch_uuid' in 'field list'

insert into
  `activity_log` (
    `log_name`,
    `properties`,
    `causer_id`,
    `causer_type`,
    `batch_uuid`,
    `subject_id`,
    `subject_type`,
    `description`,
    `updated_at`,
    `created_at`
  )
values
  (
    default,
    [ ],
    1,
    App \ Models \ Invoice,
    ?,
    1,
    App \ Models \ Customer,
    Invoice INV -000001 emailed to Customer by the system.,
    2022 -03 -21 11: 06: 16,
    2022 -03 -21 11: 06: 16
  )

/******* CONTROLLER ACTION ******************/
$user = $request->user();
$currentCompany = $user->currentCompany();

$invoice = Invoice::findOrFail($request->invoice);
 // Log the activity
activity()
            ->on($invoice->customer)
            ->by($invoice)
            ->log('Invoice :causer.invoice_number emailed to Customer by system.');

**Comment by LoganTFox** [comment]
I'm running into the same issue.

**Comment by brstreet2** [comment]
What version of spatie/laravel-activitylog are you running currently?

It seems like you have the old migration from spatie/laravel-activitylog version 3, but updated the package without running the new migration for version 4. A simple publish for the migration would fix it.

Try running this on your cmd:
php artisan vendor:publish --provider="Spatie\Activitylog\ActivitylogServiceProvider" --tag="activitylog-migrations"

You then have to migrate it in order for it to work. That's your option to make it work, or you can downgrade your activity log package from version 4 to version 3.
